src/microbe_stage/MicrobeStage.cs

```diff
@@ -51,6 +51,9 @@ public partial class MicrobeStage : CreatureStageBase<Entity, MicrobeWorldSimula
     [JsonProperty]
     private bool wonOnce;
 
+    [JsonProperty]
+    private bool tutorialEndedOnce;
```

Comment:
I think this should be documented what this does. Also from the name I thought this would be set to true when tutorial has been completed, but this seems like it is set to true when the tutorial is skipped?

Also I think there is a missing detection of immediately turning off the tutorials which should return the usual default panel configuration.

Reply:
That's fair, I could have called this "canceled" instead...